public/index.html

```diff
@@ -1,18 +1,90 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title>Climbing Card Checker</title>
-	<script async src="./app.js"></script>
-	<link rel="stylesheet" href="app.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Climbing Card Checker</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <script src="https://unpkg.com/vue@3"></script>
 </head>
+
 <body>
-	<form id="form">
-	  <label>Isikukood: <input id="idCode" type="text"></label>
-	  <br><br>
-	  <button type="submit">Kontrolli</button>
-	</form>
-	<p id="log"></p>
+    <div id="app">
+        <div id="left-background"></div>
+        <div id="left">
+            <form id="form" @submit.prevent="submit">
+                <p>Validate the climbing certifciate using the ID code</p>
```

Comment:
Let's translate everything to Estonian before the merge? That way we can give it to the reception people in reality as well.

Reply:
All the existing text is translated now if im not mistaken